docs/ldml/tr35.md

```diff
@@ -4334,13 +4334,40 @@ Other contributors to CLDR are listed on the [CLDR Project Page](https://www.uni
 
 ### Number symbols and formats without numberSystem
 - Noted that [Number Symbols](tr35-numbers.md#Number_Symbols) and [Number Formats](tr35-numbers.md#Number_Formats) without a specific `numberSystem` attribute should not be used and will be deprecated in CLDR v48.
+- 
 ### Clarified `currencyData` element ordering
 - In [Supplemental Currency Data](tr35-numbers.md#Supplemental_Currency_Data), noted that in the list of `currency` elements for a given `currencyData/region`, those with `tender="false"` should be at the end of the list.
 
 ### Semantic Datetime Skeletons
 - Added a [Time Precision](tr35-dates#Time_Precision) option, replacing the discrete time field sets.
 - Updated the algorithm for mapping to standard skeletons.
 
+### [Timezones](https://unicode.org/reports/tr35/tr35-dates.html#Using_Time_Zone_Names)  
+* Added **Specific location format**  
+* Added to clauses in formatting timezones  
+    * What to do if the canonicalization fails  
+    * How to determine location formats (generic or specific)  
+    * When to fall back to the specific location format
+   
+### [Unit Identifiers](tr35-general.html#Unit_Identifiers)
+* Modified the EBNF  
+    * Deprecated use of unit\_constants  in the numerator of units  
+    * Limited unit\_constants to 8 characters  
+    * Clarified the normalized form of units.  
+* Modified the process of formatting to handle unit\_constants  
+* [Unit Preferences Overrides](tr35-info.html#Unit_Preferences_Overrides)
+  * Clarified the process of deriving the region for overrides
+ 
+### [DTD Annotations](https://cldr.unicode.org/development/dtd_annotations)
+* Fixed the description of @ORDERED
+* Expanded the documentation for @MATCH and moved to [cldr.unicode.org](https://cldr.unicode.org/development/dtd_annotations)
+
+### [Inheritance](https://unicode.org/reports/tr35/dev/tr35.html#inheritance-marker)
```

Comment:
@macchiati why does this link to <https://unicode.org/reports/tr35/dev/tr35.html> ?

Reply:
That was to point people to the section where changes were being made, during alpha/beta. Those links all need to be fixed for release, but I don't think we had that in the BRS tasks.

Reply:
In this particular case, they should have instead been relative links, since it was within the spec! That way no update is needed.